admin/templates/configuration/etc/init.d/xrootd

```diff
@@ -50,7 +50,12 @@ export QSW_MYSQLDUMP=`which mysqldump`
 QSW_SCRATCHPATH="${QSERV_RUN_DIR}/tmp"
 QSW_SCRATCHDB="qservScratch"
 export QSW_RESULTPATH="${XROOTD_RUN_DIR}/result"
-export LSST_LOG_CONFIG="${QSERV_RUN_DIR}/etc/log4xrootd.properties"
+
+# Log configuration file for worker nodes.
+# xrootd manager doesn't use it.
+# Indeed, log4cxx is only used by Qserv plugin
+# which only runs on xrootd server side (i.e. worker nodes)
```

Comment:
That comment is somewhat confusing. We run xrootd manager (cmsd) both on worker nodes and master node. And cmsd will also use pour plugins, so it will likely have its output redirected to log4cxx as well.